c++/include/orc/Statistics.hh

```diff
@@ -321,6 +321,32 @@ namespace orc {
      * @return maximum value
      */
     virtual int64_t getMaximum() const = 0;
+
+    /**
+     * check whether column has a lowerBound
+     * @return true if column has a lowerBound
+     */
+    virtual bool hasLowerBound() const = 0;
```

Comment:
I saw these lines of changes are also in ORC-87. Is it a good idea to remove them here to make this change more concise?